pkg/config/parameters_test.go

```diff
@@ -49,6 +49,7 @@ debug: false
 kubeconfig: TestParseDefaults/.kube/config
 server:
   xds-server-type: contour
+gatewayclass-controller: projectcontour.io/projectcontour/contour
```

Comment:
Does this need to be so verbose? Can just `projectcontour.io/contour` be sufficient?

Reply:
We can remove the API group from the string to remove the verbosity.